### DISCUSSION TITLE
如何自主修改《心理学报》样式的代码，使得引用标记括号内含有页码？

### DISCUSSION THREAD

**Discussion by larppp** [zotero-chinese/wiki#329] [求助 / Q&A]
如何自主修改博主中文社区中提供的《心理学报》样式的代码，使得引用标记括号内含有具体页码呢？具体要求如下图所示：
![2024-12-21_212823](https://github.com/user-attachments/assets/0dd46733-2963-40f4-97a5-69c42dcba29d)

**Comment by northword** [comment]
https://zotero-chinese.com/user-guide/ms-word-plugin#%E9%A1%B5%E7%A0%81%E5%92%8C%E5%85%B6%E4%BB%96%E5%AE%9A%E4%BD%8D%E7%AC%A6-%E5%9B%BE%E3%80%81%E8%A1%A8%E7%AD%89

**Reply by larppp** [reply]
![image](https://github.com/user-attachments/assets/c0b11214-cf37-421c-870f-ae9c49d38e3c)
按照您的指导，操作有效，但是“p. 361”中间的空格如何删去呢？如图

**Reply by zepinglee** [reply]
APA 是要求有空格的，《心理学报》基本沿用 APA 的格式。

相关的代码位于：

https://github.com/zotero-chinese/styles/blob/c508629bd9b373ec2b10d33f17db5735062155e4/src/acta-psychologica-sinica/acta-psychologica-sinica.csl#L1206

**Reply by larppp** [reply]
谢谢，连接有效！已解决
